src/context/SyntheticsStateContext/SyntheticsStateContextProvider.tsx

```diff
@@ -236,6 +238,8 @@ export function SyntheticsStateContextProvider({
     enabled: pageType === "trade",
   });
 
+  const { blockTimestampData } = useBlockTimestamp(chainId);
```

Comment:
should we do enable this hook conditonally? e.x. not needed on leaderboard at least